### PR TITLE
feat: add responsive grid

### PR DESCRIPTION
**Pull request by JasonLantz**
I have a demo of this working on component archtype if anyone wants to see it in action.

### CODE REVIEW

**Comment by wyseguyonline** [comment]
Looks like a good candidate for the demo today.

**Comment by JasonLantz** [comment]
good call...

File: scss/mixins/_grid.scss [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    @each $breakpoint in map-keys($pe-responsive-breakpoints) {
+
+      //Get container margin by breakpoint
+      $margin                       : map-get($pe-container-margin, $breakpoint);
+      $current-breakpoint-value-get : get-breakpoint-value($breakpoint);
+
+      //Construct media query
+      @media(min-width:$current-breakpoint-value-get){
+        padding-right : ($margin / 2);
+        padding-left  : ($margin / 2);
```
Comment (parkerpearson) [inline scss/mixins/_grid.scss]: @JasonLantz the container margins listed in the design doc are intended to be on each side, not the sum of both margins. Removing the division seems to fix the horizontal scrolling issue.

**Comment by wyseguyonline** [comment]
Approved

**Comment by wyseguyonline** [comment]
Approved...pending the CI tests.

**Pull request closed without merging**